data_loader.py

```
@@ -0,0 +1,73 @@
+        # keys of data ['s', 'etyp', 'epos', 'edur', 'artifacts']
+        self.raw = self.data['s'].T
+        self.events_type = self.data['etyp'].T
+        self.events_position = self.data['epos'].T
+        self.events_duration = self.data['edur'].T
+        self.artifacts = self.data['artifacts'].T
+
+        # Types of motor imagery
+        self.mi_types = {769: 'left', 770: 'right',
+                         771: 'foot', 772: 'tongue', 783: 'unknown'}
+
+    def get_trials(self, channels=None):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_trials`

(ANN201)